Source/WebCore/platform/ScrollAnimationKeyboard.h

```diff
@@ -45,6 +45,8 @@ class ScrollAnimationKeyboard final: public ScrollAnimation {
 
     void stopKeyboardScrollAnimation();
 
+    ScrollClamping clamping() const override { return ScrollClamping::Unclamped; }
```

Comment:
Minor fears re: what this will mean for a keyboard scroll that's underway while the document changes size significantly.

Reply:
I made a test page and tested this behavior, and it works as expected / currently